CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 ## Unreleased
 
+## 0.25.0 (Oct 12, 2020)
```

Comment:
I thought this got done by the CI automatically?

Reply:
only in the consul-k8s pipeline!

Reply:
![image](https://user-images.githubusercontent.com/26226763/95791893-5a147680-0cb0-11eb-9373-3e9fd61c61fd.png)

cool

Reply:
on consul-k8s this is the case, but it is manual for consul-helm

Reply:
> (that did not show up the way i hoped i would lol)

lol it did in my email though!